src/app/person/person.component.html

```diff
@@ -1 +1,18 @@
-<div class="container"></div>
+<div class="container" *ngIf="person">
+  <h2>{{ person.first_name }} {{ person.last_name }}</h2>
+  <section>
+    <img src="{{ person.avatar.employee_big.url }}" alt="">
+    <div class="about">
+      <p>Посада: {{ person.position }}</p>
+      <!-- <p>Біографія: {{ person.biography }}</p> -->
+      <p>
+        Біографія: <span [innerHtml]="person.biography"></span>
```

Comment:
в person.biography в тебе може бути будь-що, в тому числі і p (100% там p є) - таким чином враппер для цього не підходящий, краще: 
```suggestion
        <p>Біографія:</p>
        <div [innerHtml]="person.biography"></div>
```